File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 4/ex6.c
#include <stdio.h>

main()
{
    printf("Atribuindo 0 para valores negativos de um vetor\n\n");

    //vars
    int v[40], i;

    //Entrada
    for( i=0; i<40; i++)
    {
        printf("Digite v[%d]: ", i);
        scanf("%d",&v[i]);
    }

    //Processamento
    for( i=0; i<40; i++)
    {
        if( v[i] < 0 )
            v[i] = 0;
    }

    //Saída
    printf("\n\nVetor Resultante: \n");
    for( i=0; i<40; i++)
    {
        printf("\n%d", v[i]);
    }
}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 4/src/Ex6.java
//Importando Scanner
import java.util.Scanner;

public class Ex6 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner input = new Scanner(System.in);
		
		System.out.println("Anulando Valores Negativos\n\n");
		
		//vars
		int v[] = new int[40];
		int i;
		
		//Entrada
		for( i=0; i<40; i++)
		{
			System.out.print("Digite v[" + i + "]: ");
			v[i] = input.nextInt();
		}
		
		//Processamento
		for( i=0; i<40; i++)
		{
			if( v[i] < 0)
			{
				v[i] = 0;
			}
		}
		
		//Saída
		System.out.println("\nVetor resultatante: ");
		for( i=0; i<40; i++)
		{
			System.out.printf("%d  ", v[i]);
		}
		
	}
}
